react/features/participants-pane/components/web/MeetingParticipants.tsx

```diff
@@ -109,7 +109,10 @@ function MeetingParticipants({
 
     return (
         <>
-            <div className = { styles.heading }>
+            <div
```

Comment:
Based on what you mentioned earlier, can we convert this div into a span, too?

Reply:
Hmmm the idea earlier with the chat heading was more to separate the heading text from the action button tied to it. There was no accessibility concern between using a span or a div, I guess I used a span out of habit :)

I guess here it would make more sense to leave it as a div in order to not mess with already-set CSS rules, no?

Reply:
Yes, we can.